bus/bus.go

```diff
@@ -585,6 +585,75 @@ func (b *bus) setRedundancySettings(rs RedundancySettings) error {
 	}
 }
 
+func (b *bus) paramsHandlerDownloadGET(jc jape.Context) {
+	gp, err := b.gougingParams()
+	if jc.Check("could not get gouging parameters", err) != nil {
+		return
+	}
+
+	jc.Encode(DownloadParams{
+		ContractSet:   "autopilot", // TODO
+		GougingParams: gp,
+	})
+}
+
+func (b *bus) paramsHandlerUploadGET(jc jape.Context) {
+	gp, err := b.gougingParams()
+	if jc.Check("could not get gouging parameters", err) != nil {
+		return
+	}
+
+	jc.Encode(UploadParams{
+		ContractSet:   "autopilot", // TODO
+		CurrentHeight: b.cm.TipState().Index.Height,
+		GougingParams: gp,
+	})
+}
+
+func (b *bus) paramsHandlerMigrateGET(jc jape.Context) {
+	gp, err := b.gougingParams()
+	if jc.Check("could not get gouging parameters", err) != nil {
+		return
+	}
+
+	jc.Encode(MigrateParams{
+		CurrentHeight: b.cm.TipState().Index.Height,
+		FromContracts: "", // TODO
+		ToContracts:   "", // TODO
```

Comment:
should we default this to `autopilot` as well? Same as the regular upload? 
The `FromContracts` probably still need to be a list of IDs.

Reply:
Going to change this when we add migrations, I'm not even sure why we need migrate params from the `Bus`. Shouldn't the worker be instructed to move a slab, get a set of bad hosts/contracts and then the worker should just fetch the params we'd use on a regular upload to migrate with?

Reply:
I guess I don't understand yet how the `bus` would be able to return `FromContracts`. It doesn't know that right?

Reply:
resolving this b/c I'm tackling migrations in next PR